string[] joint_names
float64[] positions
float64[] velocities
float64[] torques
float64[] acc
time stamp